src/main/resources/application.yaml

```diff
@@ -18,7 +18,7 @@ spring:
       - 'classpath:db/changelog/v.0.4/sql-import/import-form-new.sql'
       - 'classpath:db/changelog/v.0.2/sql-import/import-form-add-columns.sql'
       - 'classpath:db/changelog/v.0.3/sql-import/import-time-for-call.sql'
-      - 'classpath:db/changelog/v.0.3/sql-import/import-internship.sql'
+      - 'classpath:db/changelog/v.0.6/sql-import/import-internship.sql'
```

Comment:
Why do you change version of changelog, which already exists? What happens with previous file? I don't see any changes with it

Reply:
add update instead import